desktop/ui/src/main/java/org/datacleaner/windows/ResultWindow.java

```diff
@@ -359,15 +398,18 @@ protected JComponent getWindowContent() {
 
         final DCBannerPanel banner = new DCBannerPanel(imageManager.getImage("images/window/banner-results.png"),
                 bannerTitle);
-        _tabbedPane.bindTabTitleToBanner(banner);
+        _tabbedPane.addListener(new VerticalTabbedPane.Listener() {
+            @Override
+            public void stateChanged(int newIndex, Tab<?> newTab) {
+                banner.setTitle2(newTab.getTitle());
+                banner.updateUI();
+            }
+        });
 
         for (JComponent pluggableButton : _pluggableButtons) {
             banner.add(pluggableButton);
```

Comment:
I did see those, but I didn't see anything where those were used. Where exactly is it used in commercial editions? In terms of API to be changed, strictly speaking it wouldn't be pluggable jbuttons anymore, but pluggable jmenuitems. Also, would it make sense to redo the 'publish to server' and maybe 'export to html' buttons as a pluggable button aswell?